brambling/utils/invites.py

```diff
@@ -109,7 +110,9 @@ def send(self):
     @classmethod
     def get_invites(cls, content=None):
         kwargs = {'kind': cls.slug}
-        if content is not None:
+        if type(content) is QuerySet:
+            kwargs['content_id__in'] = [obj.pk for obj in content]
+        elif hasattr(content, 'pk'):
```

Comment:
We should either have this be `content is not None` or we should add a blanket else to throw an error if the content doesn't match expectations.